subt_ign/src/GameLogicPlugin.cc

```diff
@@ -210,7 +210,8 @@ class subt::GameLogicPluginPrivate
   public: void PublishRobotEvent(
     const ignition::msgs::Time &_simTime,
     const std::string &_type,
-    const std::string &_robot);
+    const std::string &_robot,
+    int _eventId);
```

Comment:
add `\param[in]` doxygen comment?

same for the function below

Reply:
72498e9d89c6c06ebc146e58163b8a61c6203151